com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -532,7 +532,7 @@ internal static void NetworkBehaviourUpdate()
                 else
                 {
                     // when client updates the sever, it tells it about all its objects
-                    foreach (var sobj in NetworkSpawnManager.SpawnedObjectsList)
+                    foreach (var sobj in NetworkManager.Singleton.SpawnManager.SpawnedObjectsList)
```

Comment:
```suggestion
                    var spawnedNetworkObjects = NetworkManager.SpawnManager.SpawnedObjectsList;
                    foreach (var networkObject in spawnedNetworkObjects)
```

Reply:
Ditto as above, actually making the induction variable longer doesn't to me improve readability